server/render/index.js

```diff
@@ -164,6 +165,10 @@ export function serverRender( req, res ) {
 			reduxSubtrees = cacheableReduxSubtrees;
 		}
 
+		if ( isSupportSession( req ) ) {
+			reduxSubtrees = reduxSubtrees.concat( 'support' );
```

Comment:
is the point here consistency with the above code? is it transactional swapping? it's surprising that we're not simply doing `reduxSubtrees.push( 'support' )` but not entirely distracting. just curious.

Reply:
of course, we could remove all the mutation and turn this into a single declarative stack…

```js
const isIsomorphic = isSectionIsomorphic( context.store.getState() );
const reduxSubtrees = [
	...cacheableReduxSubtrees,
	isSupportRequest( req ) && 'support',
	isIsomorphic && 'themes',
	isIsomorphic && 'ui',
].filter( a => a );
``

Reply:
> is the point here consistency with the above code? is it transactional swapping? it's surprising that we're not simply doing reduxSubtrees.push( 'support' ) but not entirely distracting. just curious.

Yeah... it was probably me half-consciously trying to use the same I found around that line, but it's silly. I'm going to change it.

Reply:
Thanks for the playing. The declarative form reads more clearly to me.